ONSdigital/eq-author
# WIP 40 add page

### What is the context of this PR?

[Trello card](https://trello.com/c/HHkiluUf/40-add-new-question-page-to-section-5)

- Added a basic sidebar with the section/page structure and an "add page" button
- Clicking "add page" button makes roundtrip to the API, optimistically updating the UI with the new page
- Section and Page fields are editable and save on blur

### How to review 

1. Navigate to an existing Q
2. Click "add page"
3. Confirm new page is added
4. Confirm focus is on the Page Title (unless the section title is empty)
5. Ensure tests pass, etc.
